src/main/java/com/box/sdk/BoxCollaboration.java

```diff
@@ -130,8 +130,14 @@ protected static BoxCollaboration.Info create(
      * @param api the API connection to use.
      * @return a collection of pending collaboration infos.
      */
-    public static Collection<Info> getPendingCollaborations(BoxAPIConnection api) {
-        URL url = PENDING_COLLABORATIONS_URL.build(api.getBaseURL());
```

Comment:
is PENDING_COLLABORATIONS_URL used now anywhere? Seems not. Constant to be removed

Reply:
This is public field. Removing it can break someone else code. I'm suggesting leaving it.